src/api/types.ts

```diff
@@ -1,11 +1,10 @@
-import { AxiosError, AxiosResponse } from "axios";
 import { DateTime } from "luxon";
 
 export interface LoginResponse {
   access_token: string;
   refresh_token: string;
   user: {
-    pk: number;
+    pk: string;
```

Comment:
The user record is not the same as the member record, this should be a number not a string